neo/SmartContract/ApplicationEngine.cs

```diff
@@ -81,6 +84,27 @@ protected override bool PreExecuteInstruction()
         {
             if (CurrentContext.InstructionPointer >= CurrentContext.Script.Length)
                 return true;
+            
+            Instruction instruction = CurrentContext.CurrentInstruction;
```

Comment:
if Trigger is `Verification` instead of `isLimited`?

Reply:
If we apply this mode only to verification, what seems to be the case, then its a nice change. You're right.